### DISCUSSION TITLE
(WEC) more info on live timing (regarding pitstops)

### DISCUSSION THREAD

**Discussion by PhillipAlpha** [f1multiviewer/issue-tracker#579] [Feature requests] [answered]
A column to show either Laps since last pitstop or time since last pitstop

ideally being able to see the length of last x (1/3/5) stints  if possible

**Comment by jeppeolesen** [comment, marked as answer]
Hey @PhillipAlpha 

Since WEC launched a new live timing product for this year, we are no longer supporting WEC timing in MultiViewer, due to the workload required to adapt the new data structure to the MultiViewer Live Timing formats.

You can still watch the FIAWEC+ streams in MultiViewer.

Closing this, as we recommend using the official WEC timing for now.

(And yes, I realise this request was made three years ago when WEC timing was still supported 😅)
